catalog/Podfile

```diff
@@ -60,7 +60,6 @@ target "MDCCatalog" do
   pod 'MaterialComponentsBeta', :path => '../', :testspecs => [
     'ActionSheet+ActionSheetThemer/UnitTests',
     'ActionSheet+ColorThemer/UnitTests',
-    'ActionSheet+Theming/UnitTests',
```

Comment:
Why are we removing the test targets?

Reply:
I put it with the component tests. Was that not what we intended?

Reply:
The theming extensions are still technically separate targets so I think it’s valuable for the test targets to match; reduces likelihood of our tests crossing concerns between separate targets.

Reply:
I can agree with that, I have reverted this change and the move.